raiden/smart_contracts/NettingChannelLibrary.sol

```diff
@@ -362,13 +364,13 @@ library NettingChannelLibrary {
         node2.netted = node2.balance + node1.transferred_amount - node2.transferred_amount;
 
         for (k = 0; k < node1.unlocked.length; k++) {
-            node1.netted += node1.unlocked[k].amount;
-            node2.netted -= node1.unlocked[k].amount;
+            node1.netted -= node1.unlocked[k].amount;
```

Comment:
One test was added that caught the error `test_unlock`, I believe we should have more tests for interwoven scenarios.